pkg/tenantNamespace/namespaceManager.go

```diff
@@ -120,3 +120,7 @@ func (nm *tenantNamespaceManager) GetNamespace(cluster discoveryv1alpha1.Cluster
 	}
 	return namespaces[0].DeepCopy(), nil
 }
+
+func getNameForNamespace(cluster discoveryv1alpha1.ClusterIdentity) string {
+	return fmt.Sprintf("liqo-tenant-%s", cluster.ClusterName)
```

Comment:
Should this be constructed from the unique name?

Reply:
Yes, good catch - the tenant ns may be created as part of the auth process, when the cluster name is not guaranteed to be unique.